src/platform/user/profile/vet360/components/PhoneField/VAPPhoneField.jsx

```diff
@@ -137,17 +137,24 @@ export default class PhoneField extends React.Component {
   };
 
   render() {
-    const ContentView =
-      this.props.fieldName === FIELD_NAMES.MOBILE_PHONE
-        ? ({ data }) => {
-            return (
-              <div>
-                <VAPPhoneView data={data} />
-                <ReceiveTextMessages fieldName={FIELD_NAMES.MOBILE_PHONE} />
-              </div>
-            );
-          }
-        : VAPPhoneView;
+    const ContentView = ({ data }) => {
+      const { areaCode, phoneNumber, extension } = data;
+
+      return (
+        <>
+          <Telephone
+            contact={`${areaCode}${phoneNumber}`}
+            extension={extension}
+            className="non-clickable-link"
```

Comment:
Maybe follow up this PR with a change to the `Telephone` component that disables clickability via a prop?

Reply:
Exactly what I was thinking! 💯